strands_navfn/package.xml

```diff
@@ -0,0 +1,52 @@
+<package>
+    <name>strands_navfn</name>
+    <version>1.12.2</version>
```

Comment:
This will fail, as this package number doesn't match the others in this repo. Should be `0.0.18`